object_storage_api/stores/image.py

```diff
@@ -37,3 +39,22 @@ def upload(self, image_id: str, image_metadata: ImagePostMetadataSchema, upload_
         )
 
         return object_key
+
+    def create_presigned_get(self, image: ImageOut) -> HttpUrl:
+        """Generate a presigned URL to share an S3 object.
+
+        :param image: `ImageOut` model of the image.
+        :return: Image metadata with a presigned url.
+        """
```

Comment:
```suggestion
    def create_presigned_get(self, image: ImageOut) -> str:
        """
        Generate a presigned URL to get an S3 object.

        :param image: `ImageOut` model of the image.
        :return: Image metadata with a presigned url.
        """
```
I tried printing out the type(response) and it was a str instead of HttpUrl.